playbooks/roles/plugins/remote_execution/tasks/main.yml

```diff
@@ -0,0 +1,7 @@
+- name: 'Install remote execution plugin'
+  shell: >
+    foreman-installer
+    --enable-foreman-plugin-remote-execution
+    --enable-foreman-proxy-plugin-remote-execution-ssh
```

Comment:
General comment, one nice thing about the foreman_installer role is you can pass any options to it, for example if I wanted to be able to configure any of the options for a plugin.